CollAction/Program.cs

```diff
@@ -3,22 +3,24 @@
 using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
+using CollAction;
+using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
 
-namespace CollAction
+namespace WebApplication2
```

Comment:
Strange namespace...

Reply:
This was corrected in a later commit. You can look at the "Changes" tab for the effective changes.